cli/azd/pkg/environment/data_store.go

```diff
@@ -0,0 +1,38 @@
+package environment
+
+import (
+	"context"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/contracts"
+)
+
+// DataStore is the interface for the interacting with the persistent storage of environments.
+
+type RemoteKind string
+
+const (
+	RemoteKindAzureStorage RemoteKind = "AzureStorage"
+)
+
+type DataStore interface {
+	// Gets the path to the environment .env file
+	Path(env *Environment) string
```

Comment:
Nit: Consider adding a prefix like `Env` so this is `EnvPath`.  It would provide symmetry with `ConfigPath` and make it a little clearer, I think, about what actually is being returned.